### PR TITLE
Adding Radis 

### PR DESCRIPTION
**Pull request by arunavabasu-03**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by arunavabasu-03** [comment]
@conda-forge-linter it is ready to review !!

**Comment by arunavabasu-03** [comment]
@conda-forge-admin. , it is ready to review !!

File: recipes/radis/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    - progressbar2 # used in vaex
+    - psutil # for getting user RAM
+    - pytest #testing
+    - scipy >=1.4.0
+    - seaborn # other matplotlib themes
+    - termcolor # terminal colors
+    - vaex >=4.9.2 # load HDF5 files
+    - hjson-py # Json with comments (for default_radis.json) 
+    - pytables # for pandas to HDF5 export
+    - json_tricks # to deal with non jsonable formats
```
Comment (ocefpaf) [inline recipes/radis/meta.yaml:51]: All the comments here seems to belong in an upstream documentation. They are not very helpful here in the package.

File: recipes/radis/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,86 @@
+{% set name = "radis" %}
+{% set version = "0.13.1" %}
```
Comment (ocefpaf) [inline recipes/radis/meta.yaml:2]: Latest seems to be 0.14. Any particular reason to add an older one?

File: recipes/radis/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 650ef57b6f6b96232e4282564b613650538ff914268e71b67a4ea846c9a8d746
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
```
Comment (ocefpaf) [inline recipes/radis/meta.yaml:18]: Upstream seem to have a compiled extension and other other dependencies that you missed there. Can you try to generate the recipe with `grayskull pypi --strict-conda-forge radis` instead? It is much less error prone and easier for us to review.

**Comment by jakirkham** [comment]
@conda-forge-admin, please ping team

**Comment by arunavabasu-03** [comment]
@ocefpaf can you take a look into this !! I fix as per your reviews 😀!!

**Comment by arunavabasu-03** [comment]
@conda-forge-admin, please ping team

**Comment by ocefpaf** [comment]
Looks like there is a missing dependency on Windows:

```
- Encountered problems while solving:
-   - package vaex-4.11.0-pyhd8ed1ab_0 requires vaex-core <4.12.0,>=4.11, but none of the providers can be installed
```

If that is not fixable you may want to skip Windows to avoid the failing CIs.

**Comment by arunavabasu-03** [comment]
@conda-forge-admin, please ping team

File: recipes/radis/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+    - lmfit
+    - {{ pin_compatible('numpy') }}
+
+test:
+  imports:
+    - radis
+
+about:
+  home: https://github.com/radis/radis
+  license: LGPL-2.0-only
```
Comment (dopplershift) [inline recipes/radis/meta.yaml:62]:
```suggestion
  license: LGPL-3.0-only
```

**Comment by arunavabasu-03** [comment]
@conda-forge-admin, please ping team !!

**Comment by arunavabasu-03** [comment]
@conda-forge-admin, please ping team !!

**Comment by arunavabasu-03** [comment]
@conda-forge-admin, please ping team !!

**Comment by arunavabasu-03** [comment]
@conda-forge-admin, please ping team !!

**Review by dopplershift — APPROVED** [review]

**Pull request merged**